docs/changelog/132162.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 132162
```

Comment:
Do we want both changelog entries on this branch?

Reply:
we do not wan both. will remove the backport id one.

Reply:
deleted by [32c0b62](https://github.com/elastic/elasticsearch/pull/132172/commits/32c0b62d434150d74d732d64c3742d4a8bd7854b)